var/spack/repos/builtin/packages/odgi-git/package.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class OdgiGit(CMakePackage):
```

Comment:
I disagree that this should be called odgi-git. I would prefer just 'odgi', matching the tool and the git name.

I want to depend on this by the name 'odgi' or 'odgi@version'. Having an odgi and an odgi-git package means I have to be aware of the two packages and change the name when chaging from a git version to a non-git version. This can easily happen when odgi does a proper release version.

Reply:
Just so you know, when I created a package from the git repository, it created a '-git' suffixed package.

Reply:
I am doing as you said